code/game/machinery/telecomms/machine_interactions.dm

```diff
@@ -123,14 +123,13 @@
 			if(params["value"])
 				tempfreq = text2num(params["value"]) * 10
 		if("freq")
-			var/newfreq = tempfreq * 10
-			if(newfreq == FREQ_SYNDICATE)
-				to_chat(operator, "<span class='warning'>Error: Interference preventing filtering frequency: \"[newfreq / 10] GHz\"</span>")
+			if(tempfreq == FREQ_SYNDICATE)
+				to_chat(operator, "<span class='warning'>Error: Interference preventing filtering frequency: \"[tempfreq / 10] kHz\"</span>")
```

Comment:
```suggestion
				to_chat(operator, "<span class='warning'>Error: Interference preventing filtering frequency: \"[tempfreq] kHz\"</span>")
``` old code expected a number 10 times as high

Reply:
it's good. tempfreq holds 1213, gotta divide by 10 to get 121.3 for the UI